dora/core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -953,10 +954,10 @@ public List<BlockWorkerInfo> getAllWorkers() throws IOException {
     try (ReinitBlockerResource r = blockReinit()) {
       // Use membership mgr
       if (mMembershipManager != null && !(mMembershipManager instanceof MasterMembershipManager)) {
-        List<BlockWorkerInfo> liveWorkers = mMembershipManager.getLiveMembers().stream()
+        List<BlockWorkerInfo> liveWorkers = Streams.stream(mMembershipManager.getLiveMembers())
```

Comment:
what does this lib do better than java stream?

Reply:
`mMembershipManager.getLiveMembers()` returns `WorkerClusterView` which is an `Iterable<WorkerInfo>` and `Iterable`s does not implement `Stream`. `Streams.stream()` is an adapter that converts an `Iterable` to a `Stream`.

Reply:
I added a `.stream()` method on `WorkerClusterView` so that we don't need another dependency.